packages/compiler/src/core/messages.ts

```diff
@@ -576,6 +576,12 @@ const diagnostics = {
       default: "The #deprecated directive cannot be used more than once on the same declaration.",
     },
   },
+  unnecessary: {
```

Comment:
Nit: Should this be named `unused`? Does it make sense to have import/using - specific messages?

I prefer `unused` over `unnecessary` since the latter has a subjective connotation, and at least in TS, `unused` is common for linter rules.

Reply:
I am fine with "unused", update "unnecessary" to "unused-import" and "unused-using". thanks.